xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-6.33.0
+6.33.1
```

Comment:
Why? Current master is at 6.33.0. I know that there are other PRs that touch this version but the order of merge will decide which PR will have to bump to which version.